src/transactionsFactories/accountTransactionsFactory.ts

```diff
@@ -3,7 +3,7 @@ import { IAddress } from "../interface";
 import { Transaction } from "../transaction";
 import { TransactionBuilder } from "./transactionBuilder";
 
-interface IConfig {
+export interface IConfig {
```

Comment:
I don't think this should be exported. If we plan on using the same interfaces throughout the package (e.g. controllers) we should have an `interfaces.ts` file. If it's used in other places we should simply duplicate the inteface.